partials/footer.hbs

```diff
@@ -0,0 +1,74 @@
+<div class="grid grid-cols-6 px-16 py-12 bg-black text-white text-sm">
```

Comment:
This can be made into a `<footer>` element instead, to be more semantically accurate.